Dockerfile

```
@@ -59,8 +60,16 @@ RUN uv sync --locked \
   && if [[ -n "${OPENSHIFT_PYTHON_WRAPPER_COMMIT}" ]]; then uv pip install git+https://github.com/RedHatQE/openshift-python-wrapper.git@$OPENSHIFT_PYTHON_WRAPPER_COMMIT; fi \
   && if [[ -n "${OPENSHIFT_PYTHON_UTILITIES_COMMIT}" ]]; then uv pip install git+https://github.com/RedHatQE/openshift-python-utilities.git@$OPENSHIFT_PYTHON_UTILITIES_COMMIT; fi \
   && if [[ -n "${TIMEOUT_SAMPLER_COMMIT}" ]]; then uv pip install git+https://github.com/RedHatQE/timeout-sampler.git@$TIMEOUT_SAMPLER_COMMIT; fi \
+  && if [[ -n "${PYHELPER_UTILS_COMMIT}" ]]; then uv pip install git+https://github.com/RedHatQE/pyhelper-utils.git@$PYHELPER_UTILS_COMMIT; fi \
   && rm -rf ${TEST_DIR}/.cache \
   && rm -rf ${TEST_DIR}/artifacts \
   && find ${TEST_DIR}/ -type d -name "__pycache__" -print0 | xargs -0 rm -rfv
 
+# CoverPort runtime coverage instrumentation
+COPY coverport/ /app/coverport/
+RUN pip install --no-cache-dir coverage>=7.0
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)